sql/core/src/test/resources/sql-tests/results/number-format.sql.out

```diff
@@ -0,0 +1,50 @@
+-- Automatically generated by org.apache.spark.sql.SQLQueryTestSuite
+-- Number of queries: 4
+
+
+-- !query 0
+select 1, -1
+-- !query 0 schema
+int, int
+-- !query 0 output
++---+----+
+|  1|(-1)|
```

Comment:
Putting the result of `showString` directly in golden file looks not good:

1. the ornamental `+----+` is useless and could be bad if we wanna do side by side string comparison.
2. the column names should be part of the query schema, not output rows

We need a way to encode output data into string and parse it back. A simple way is to use `\t` to separate each column, but not sure if it's good enough, do you know some existing framework that we can follow?